packages/larva-tokens/package.json

```
@@ -18,13 +18,14 @@
   ],
   "scripts": {
     "test": "echo \"Error: run tests from root\" && exit 1",
-    "build": "../../node_modules/.bin/gulp"
+    "build": "./node_modules/.bin/gulp"
```

```suggestion
    "build": "gulp"
```

npm should resolve this to the local copy automatically. Was that not happening?

---

Fixed in main branch - thanks!